source/channels/make-calls.rst

```diff
@@ -16,14 +17,16 @@ Mattermost introduces beta voice calling and screen sharing functionality in cha
 
 - `Debugging <#debugging>`__
 
-Features available in Mattermost 7.0
-------------------------------------
+Features 
+--------
 
-From 7.0, you can:
+Starting from Mattermost v7.0, you can:
```

Comment:
Call recordings will be available from v7.6 (self-hosted only). May be good to specify it below.